lib/ddtrace/span.rb

```diff
@@ -92,12 +102,15 @@ def clear_tag(key)
 
     # Return the tag with the given key, nil if it doesn't exist.
     def get_tag(key)
-      @meta[key]
+      @meta[key] || @metrics[key]
```

Comment:
this is so weird to read for me, in Python empty string is falsey

Reply:
Only false values are `false` and `nil` in Ruby. Everything else evaluates to `true`.